lib/perl/Genome/InstrumentData/Command/Import/Basic.pm

```diff
@@ -87,12 +87,15 @@ sub execute {
 sub _resolve_work_flow_inputs {
     my $self = shift;
 
-    return Genome::InstrumentData::Command::Import::WorkFlow::Inputs->create(
-        analysis_project => $self->analysis_project,
-        library => $self->library,
-        source_files => [ $self->source_files ],
-        instrument_data_properties => $self->_resolve_instrument_data_properties,
-    );
+    my $factory = Genome::InstrumentData::Command::Import::Inputs::Factory->get;
+    $factory->analysis_project($self->analysis_project);
```

Comment:
@tmooney I'm SURE I had a reason :smile: Let me review it and see if it is necessary.